src/main/asciidoc/_chapters/ops_mgt.adoc

```diff
@@ -1904,14 +1904,20 @@ It is also prepended with identifying tags `(responseTooSlow)`, `(responseTooLar
 
 ==== Configuration
 
-There are two configuration knobs that can be used to adjust the thresholds for when queries are logged.
+There are four configuration knobs that can be used to adjust the thresholds for when queries are logged.
```

Comment:
Can you add 1-2 more sentences here explaining why Scan gets its own? For example "Two knobs allow controlling the size and time threshold for all queries. Since Scans can often be larger and slower than other queries, there is an additional two knobs for setting a different threshold for those."